src/SbManager/Content/tmpl/directives/peek.html

```diff
@@ -6,13 +6,14 @@ <h3 class="panel-title">Peek {{type}}</h3>
             </div>
             <div class="panel-footer">
                 <strong>Top: </strong>
-                <input ng-model="peekCount" class="form-control peek-btn" type="number" />
+                <input ng-model="peekCount" class="form-control peek-btn" type="number"/>
                 <button ng-click="peek(peekCount)" type="button" class="btn btn-info">Peek</button>
+                <input ng-show="messages.length > 0" ng-model="searchText" placeholder="Filter messages" class="form-control search-btn" type="text" title="keyword filter match the content of the message bodies (case insensitive)"/>
```

Comment:
Should the title attribute read "keyword filter matches the content of the message body (case insensitive)"?